test/functional/wallet_upgradetohd.py

```diff
@@ -190,8 +190,8 @@ def run_test(self):
         node.stop()
         node.wait_until_stopped()
         self.start_node(0, extra_args=['-rescan'])
-        assert_raises_rpc_error(-13, "Error: Please enter the wallet passphrase with walletpassphrase first.", node.upgradetohd, mnemonic)
-        assert_raises_rpc_error(-14, "The wallet passphrase entered was incorrect", node.upgradetohd, mnemonic, "", "wrongpass")
+        assert_raises_rpc_error(-4, "Wallet encrypted but passphrase not supplied. Please refer to \"help upgradetohd\" for guidance.", node.upgradetohd, mnemonic)
```

Comment:
@thephez do we ever tell users to look at help in error messages?

Reply:
Not really. Running a command (that requires parameters) will automatically show help. Or using the Qt console tells people to type help for an overview (plus autocompletes help for messages). From CLI, using `-help` provides help which is natural to anyone working on cli.

Reply:
The problem here is we need users to supply the passphrase to `upgradetohd`'s argument named `walletpassphrase` ,  not to be confused with the RPC `walletpassphrase`, which doesn't work for `upgradetohd` but is also usually what comes to mind when hearing of `walletpassphrase`. What phrasing would convey that the best?